packages/salesforcedx-vscode-core/resources/dep.svg

```diff
@@ -0,0 +1 @@
+<svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24"><path d="M12 5c3.453 0 5.891 2.797 5.567 6.78 1.745-.046 4.433.751 4.433 3.72 0 1.93-1.57 3.5-3.5 3.5h-13c-1.93 0-3.5-1.57-3.5-3.5 0-2.797 2.479-3.833 4.433-3.72-.167-4.218 2.208-6.78 5.567-6.78zm0-2c-4.006 0-7.267 3.141-7.479 7.092-2.57.463-4.521 2.706-4.521 5.408 0 3.037 2.463 5.5 5.5 5.5h13c3.037 0 5.5-2.463 5.5-5.5 0-2.702-1.951-4.945-4.521-5.408-.212-3.951-3.473-7.092-7.479-7.092z"/></svg>
```

Comment:
Where did we get this svg from ? I want to know if we need to add any license reference to the core package.

Reply:
I'm updating this to use an icon from the Lightning Design System. Will we need to add any license reference in that case?

Reply:
Why do we want an icon from lds ? In this type of scenario we try to use the icons shipped as part of vscode (octicons)

Reply:
The octicons actually only have cloud upload & cloud download options: 
![Screen Shot 2019-05-07 at 11 25 46 AM](https://user-images.githubusercontent.com/21326913/57323855-6bb91e00-70bb-11e9-9b75-26a4c4c7a15a.png)
![Screen Shot 2019-05-07 at 11 27 27 AM](https://user-images.githubusercontent.com/21326913/57323857-6eb40e80-70bb-11e9-9982-edaa800a7ff9.png)

I thought that might be confusing because users are able to do deploy & retrieve with the org browser/working set so I went with just the cloud icon from lds:
![Screen Shot 2019-05-07 at 11 28 26 AM](https://user-images.githubusercontent.com/21326913/57323931-9c00bc80-70bb-11e9-98de-6e48686c0992.png)
I'm open to using whichever one looks best though!

Reply:
The Salesforce icon is best I think for this case.

Reply:
Cool, I'll leave it as is then.